packages/frontend/src/pages/Loading.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+import { FullWidthText } from "../components/Input/Text";
+
+const Loading = () => {
```

Comment:
Can you use the same format of `export function` that we have in the other files?